components/Header/constants.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Defines the links to be show on the header navbar
+ */
+export const headerNavbarLinks = [
+  {
+    label: "Ações",
+    sectionId: "actions-section",
```

Comment:
It should be extracted to the structure of the sections since the id of them shouldn't be defined here. 

Therefore, this PR has nothing to do with sections and we need to implement the scroll functionality on the links - Let's just leave it as a todo here for now.

Reply:
Added a todo comment on 3f1e435573920437f0dc9d857b1554d2e4f32452